vertical-pod-autoscaler/enhancements/4016-in-place-updates-support/README.md

```diff
@@ -229,8 +229,6 @@ tested in the following scenarios:
 
 * Admission controller applies recommendation to pod controlled by VPA. 
 * In-place update applied to all containers of a pod.
-* Partial updates applied to some containers of a pod, some changes skipped (request in
```

Comment:
Can we still mention this point somewhere in this AEP? Maybe in "Details still to consider" section? So this doesn't get lost. It'll be great to implement partial updates at some point :)